outline (#1)
ultralight-beam/UBIPs: README.md

# Proposing Changes to Ultralight Beam ("UB")

> ## Introduction

Reviewer: ```suggestion

## Introduction
```

> The UB project's development process is design-driven. Significant changes to the language, libraries, or tools must be first discussed, and sometimes formally documented, before they can be implemented.

Reviewer: ```suggestion
The UB project's development process is design-driven. Significant changes to the protocol, libraries, or tools must be first discussed, and sometimes formally documented, before they can be implemented.
```

This document describes the process for proposing, documenting, and implementing changes to the UB project.

## The Proposal Process
The proposal process is the process for reviewing a proposal and reaching a decision about whether to accept or decline the proposal.

1. The proposal author creates a brief issue describing the proposal.
Note: There is no need for a design document at this point.
Note: A non-proposal issue can be turned into a proposal by simply adding the proposal label.

2. A discussion on the issue tracker aims to triage the proposal into one of three outcomes:
  - Accept proposal, or
  - decline proposal, or
  - ask for a design doc.
 
  If the proposal is accepted or declined, the process is done. Otherwise the discussion is expected to identify concerns that should be addressed in a more detailed design.

3. The proposal author writes a design doc to work out details of the proposed design and address the concerns raised in the initial discussion.

4. Once comments and revisions on the design doc wind down, there is a final discussion on the issue, to reach one of two outcomes:
  - Accept proposal or

>   - decline proposal.

Reviewer 2: Formatting not consistent from point 2.

Author: fixed

# Proposing Changes to Ultralight Beam ("UB")
## Introduction
The UB project's development process is design-driven. Significant changes to the protocol, libraries, or tools must be first discussed, and sometimes formally documented, before they can be implemented.

This document describes the process for proposing, documenting, and implementing changes to the UB project.

## The Proposal Process

> 1. The proposal author [creates a brief issue](https://golang.org/issue/new) describing the proposal.\

Reviewer: ```suggestion
1. The proposal author [creates a brief issue](https://github.com/ultralight-beam/UBIPs/issues/new) describing the proposal.\
```

   Note: There is no need for a design document at this point.\
   Note: A non-proposal issue can be turned into a proposal by simply adding the proposal label.

2. A discussion on the issue tracker aims to triage the proposal into one of three outcomes:
     - Accept proposal, or
     - decline proposal, or
     - ask for a design doc.

   If the proposal is accepted or declined, the process is done.
   Otherwise the discussion is expected to identify concerns that
   should be addressed in a more detailed design.

3. The proposal author writes a design doc (TBD) to work out details of the proposed
   design and address the concerns raised in the initial discussion.

4. Once comments and revisions on the design doc wind down, there is a final
   discussion on the issue, to reach one of two outcomes:

>     - Accept proposal or

Reviewer 2: add a comma. otherwise, LGTM

Author: suggest it plz

Reviewer 2: ```suggestion
    - Accept proposal, or
```

This document describes the process for proposing, documenting, and implementing changes to the UB project.

## The Proposal Process
1. The proposal author [creates a brief issue](https://golang.org/issue/new) describing the proposal.\
   Note: There is no need for a design document at this point.\
   Note: A non-proposal issue can be turned into a proposal by simply adding the proposal label.

2. A discussion on the issue tracker aims to triage the proposal into one of three outcomes:
     - Accept proposal, or
     - decline proposal, or
     - ask for a design doc.

   If the proposal is accepted or declined, the process is done.
   Otherwise the discussion is expected to identify concerns that
   should be addressed in a more detailed design.

> 3. The proposal author writes a [design doc](#design-documents) to work out details of the proposed

Reviewer: Referencing a non existant section
